admin/aadNestedTemplate.md

```diff
@@ -0,0 +1,302 @@
+| `aadsPublicIP` | The IP address of the LDAP server |
+| `aadsServerHost` | The hostname of the Active Directory Domain Services server. |
+| `adminVMName`| At deployment time, if this value was changed from its default value, the value used at deployment time must be used.  Otherwise, this parameter should be omitted. |
+| `location` | Must be the same region into which the server was initially deployed. |
+| `wlsDomainName` | The name of the {{ site.data.var.wlsFullBrandName }} domain. |
+| `wlsLDAPGroupBaseDN` | The base distinguished name (DN) of the tree in the LDAP directory that contains groups. |
+| `wlsLDAPPrincipalPassword` | The credential (usually a password) used to connect to the LDAP server. |
+| `wlsLDAPPrincipal` | The Distinguished Name (DN) of the LDAP user that {{ site.data.var.wlsFullBrandName }} should use to connect to the LDAP server. |
+| `wlsLDAPProviderName` | (optional) The value used for creating authentication provider name of WebLogic Server. |
+| `wlsLDAPSSLCertificate` | Client certificate that will be imported to trust store of SSL. |
+| `wlsLDAPSSLCertificate` | See below for details. |
+| `wlsLDAPUserBaseDN` | The base distinguished name (DN) of the tree in the LDAP directory that contains users. |
+| `wlsPassword` | Must be the same value provided at deployment time. |
+| `wlsUserName` | Must be the same value provided at deployment time. |
+
+### `_artifactsLocation`
+
+This value must be the following.
+
+```bash
+{{ armTemplateBasePath }}
+```
+
+### `wlsLDAPSSLCertificate`
+
+Use base64 to encode your existing SSL certificate.
+
+```bash
+base64 your-certificate.cer -w 0 >temp.txt
+```
+
+Use the content as this file as the value of the `wlsLDAPSSLCertificate` parameter.
+
+#### Example Parameters JSON
+
+Here is a fully filled out parameters file.   Note that we did not include values for parameters that have a default value.
+
+```json
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "_artifactsLocation": {
+            "value": "{{ armTemplateBasePath }}"
+        },
+        "aadsPublicIP": {
+            "value": "1.2.3.4"
+        },
+        "aadsServerHost": {
+           "value": "ladps.fabrikam.com"
+        },
+        "location": {
+            "value": "eastus"
+        },
+        "wlsDomainName": {
+          "value": "adminDomain"
+        },
+        "wlsLDAPGroupBaseDN": {
+            "value": "OU=AADDC Users,DC=fabrikam,DC=com"
+        },
+        "wlsLDAPPrincipal": {
+            "value": "CN=WLSTest,OU=AADDC Users,DC=fabrikam,DC=com"
+        },
+        "wlsLDAPPrincipalPassword": {
+            "value": "Secret123!"
+        },
+        "wlsLDAPSSLCertificate": {
+            "value": "MIIKQQIBAz....EkAgIIAA=="
+        },
+        "wlsLDAPUserBaseDN": {
+            "value": "OU=AADDC Users,DC=fabrikam,DC=com"
+        },
+        "wlsPassword": {
```

Comment:
Ugh... We're probably going to get called out for hardcoded passwords again. Does this have to be here?

Reply:
comment tag: hardcodedPassword.

Yes, it does, because it is the way things work when invoking ARM templates from the command line.  However, the password itself is not passed on the command line.  Rather, the password is stored in the `parameters.json` file.